test.properties

```diff
@@ -9563,9 +9563,9 @@
     test.batch.dist.app.servers[smoke]=tomcat
 
     test.batch.names[smoke]=\
-        functional-smoke-tomcat90-mysql57
+        functional-smoke-tomcat-mysql57-jdk17_zulu
```

Comment:
@vicnate5 is this jdk17_zulu syntax correct? seems to not take it and still used jdk8 on https://testray.liferay.com/#/project/35392/routines/45357/build/35860829

Reply:
Please check with CI team